app/actions/handlers.py

```diff
@@ -0,0 +1,216 @@
+import asyncio
+import datetime
+import httpx
+import logging
+import stamina
+import app.actions.client as client
+
+from datetime import timezone
+
+from app.actions.configurations import AuthenticateConfig, PullEventsConfig
+from app.services.activity_logger import activity_logger
+from app.services.gundi import send_events_to_gundi
+from app.services.state import IntegrationStateManager
+
+
+GFW_GLAD_ALERT = "gfwgladalert"
+GFW_FIRE_ALERT = "gfwfirealert"
+
+
+logger = logging.getLogger(__name__)
+state_manager = IntegrationStateManager()
+
+
+async def handle_transformed_data(transformed_data, integration_id, action_id):
+    async for attempt in stamina.retry_context(
+            on=httpx.HTTPError,
+            attempts=3,
+            wait_initial=datetime.timedelta(seconds=10),
+            wait_max=datetime.timedelta(seconds=10),
+    ):
+        with attempt:
+            try:
+                response = await send_events_to_gundi(
+                    events=transformed_data,
+                    integration_id=integration_id
+                )
+            except httpx.HTTPError as e:
+                msg = f'Sensors API returned error for integration_id: {integration_id}. Exception: {e}'
+                logger.exception(
+                    msg,
+                    extra={
+                        'needs_attention': True,
+                        'integration_id': integration_id,
+                        'action_id': "pull_events"
+                    }
+                )
+                return [msg]
+            else:
+                """
+                for vehicle in transformed_data:
+                    # Update state
+                    state = {
+                        "latest_device_timestamp": vehicle.get("recorded_at")
+                    }
+                    await state_manager.set_state(
+                        str(integration.id),
+                        "pull_observations",
+                        state,
+                        vehicle.get("source")
+                    )
+                """
```

Comment:
Let's remove this.

Reply:
But we are going to save this:

```
{
  "gfw_integrated_alerts": {
    "dataset": "gfw_integrated_alerts",
    "latest_updated_on": "2024-05-08T15:33:47.485916+00:00",
    "version": "v20240211"
  },
  "latest_cartodb_id": 0
}
```

for the integrated alerts right? That's why I left that commented